### ISSUE TITLE
Spectating Bugs (nightly 0.1.14.670) RacerX viewing - BCSpazer hosting

### ISSUE THREAD

**Issue by RacerXx** [LunaMultiplayer/LunaMultiplayer#87]
Remember to attach the "KSP.log" and the "output_log.txt" whenever is possible!  
You can find the KSP.log in ".\Kerbal Space Program\KSP.log"  
And output_log.txt in ".\Kerbal Space Program\KSP_x64_Data\output_log.txt"

In case you are using a nighly build of LMP from appveyor include the build number.
For example: 0.0.9.**384**
Otherwise, specify the version of LMP you are using from the github releases.

**If you have access to the server. Please zip the whole "Universe" and "Config" folder.
Also if you are running mods in the server, attach the "LMPModControl.txt"**

------------------------------------------------------------------------------
------------------------------------------------------------------------------
------------------------------------------------------------------------------
**LOGS** 
[Spectate_Test.zip](https://github.com/LunaMultiplayer/LunaMultiplayer/files/1746208/Spectate_Test.zip)
[output_log_RacerX.zip](https://github.com/LunaMultiplayer/LunaMultiplayer/files/1746215/output_log_RacerX.zip)

OS: 
**Windows 10**
------------------------------------------------------------------------------

KSPVersion: 
1.3.1
------------------------------------------------------------------------------

LMP version (include the build number if you are using a nighly build): 
0.1.14.670 nightly
------------------------------------------------------------------------------

MODS (add link to download them): 
Mechjeb (RacerX hyperedit loaded not used)
------------------------------------------------------------------------------

Expected behaviour: 
Spectating
------------------------------------------------------------------------------

Current behaviour: 
Intermittent anomalies 
-----------------------------------------------------------------------------

Steps to reproduce: 
1: sync caused a explosion of vessel but then recovered as normal on viewer side (somewhere between atmosphere and orbit transition
2: around the mun in orbit. Saw the engine blow up and several explosions during staging events which in some cases created ghost ships and bits of the destroyed copy on all clients screens and tracking stations.
3: Vessel being handed off to the person viewing after drop ship seperated and became 2 parts. I got one part as a viewer, the other person got the other part of the ship (could be handy for a co-op mode? toggle option?) But then it started to just hand off the vessel to the viewer for no apparent reason after that.
4. A second viewer came online and was viewing also. So 2 viewers on same ship. Was this Anticipated? Don't really know the results from that as the bugs were already happening at the time. Unable to recreate that scenario again at this time
Long story short: Start with the staging issue then retest the vessel handoff to other clients. Everything was perfect until in-orbit staging. Atmosphere staging was good although dekessler deleted the SRB's as soon as they detatched. Viewing the SRB's falling away was not possible. (part deleted)
